nucleus/admin/cli/src/main/java/com/sun/enterprise/admin/cli/LoginCommand.java

```diff
@@ -120,21 +121,32 @@ protected int executeCommand() throws CommandException, CommandValidationExcepti
      * Prompt for the admin user name.
      */
     private String getAdminUser() {
-        Console cons = System.console();
         String user = null;
-        String defuser = programOpts.getUser();
-        if (defuser == null) {
-            defuser = SystemPropertyConstants.DEFAULT_ADMIN_USER;
-        }
-        if (cons != null) {
-            cons.printf("%s", strings.get("AdminUserPrompt", defuser));
-            String val = cons.readLine();
-            if (val != null && val.length() > 0){
-                user = val;
-            } else {
-                user = defuser;
+
+        try (ConsoleReader cons = new ConsoleReader(System.in, System.out, null)) {
```

Comment:
`cons` should be `console`

Reply:
I was just keeping it the same, but agreed I'll change it